skorch/utils.py

```diff
@@ -466,3 +466,16 @@ def get_map_location(target_device, fallback_device='cpu'):
             ), DeviceWarning)
         map_location = torch.device(fallback_device)
     return map_location
+
+
+def set_optimizer_param(optimizer, param_group, param_name, value):
```

Comment:
Would this function not be a better fit for `setter.py`?

Reply:
I wouldn't leave the setter exposed as a public function in `setter.py` since it might become confusing what a 'setter' is when mixing `set_` and `thing_setter` functions. Therefore I marked it as protected and moved it there to avoid this confusion.